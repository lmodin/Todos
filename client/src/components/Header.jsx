import React from 'react';

const Header = (props) => {

  return (
    <div className="header" >Your list of things to do</div>
  )
}

export default Header