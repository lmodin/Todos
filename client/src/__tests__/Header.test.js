import React from 'react';
//import { cleanup, fireEvent, render} from '@testing-library/react';
import Header from '../components/Header.jsx';
import renderer from 'react-test-renderer';


//snapshot testing
it('renders correctly', () => {
  const header = renderer
    .create(<Header></Header>)
    .toJSON();
  expect(header).toMatchSnapshot();
})