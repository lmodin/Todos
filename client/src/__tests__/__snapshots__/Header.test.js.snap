// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header"
>
  Your list of things to do
</div>
`;
